src/app/templates/app/convert.html

```diff
@@ -138,41 +133,52 @@ <h4 id="doclabel">Upload SPDX Document using the button or by dragging and dropp
   $("#convertpage").addClass('linkactive');
 
   $("#from_format").change(function() {
-    $("#tagToRdfFormat").prop("disabled",'disabled');
-    $(".tagToRdfOptions").css("color",'grey');
     if ($(this).data('options') === undefined) {
       /*Taking an array of all options-2 and kind of embedding it on the select1*/
       $(this).data('options', $('#to_format option').clone());
     }
     var id = $(this).val();
-    var options = $(this).data('options').filter('[data-value=' + id + ']');
+	var id2 = $('#to_format').val();
+    var options = $(this).data('options').filter('[data-value!=' + id + ']');
     $('#to_format').html(options);
     if (id!="0"){
       $('#doclabel').text("Upload SPDX "+ id +" Document using the button or by dragging and dropping onto the dashed region");
     }
     else{
       $('#doclabel').text("Upload SPDX Document using the button or by dragging and dropping onto the dashed region");
     }
-    var id2 = $('#to_format').val();
-    if (id2=="Spreadsheet"){
+	if (id2 == id) {
+	    $('#to_format').val("0");
+    } else {
+		$('#to_format').val(id2);
+	}
```

Comment:
i think, we don't need else block, since `id2=$('#to_format').val();`

Reply:
Fixed in my commit.

Reply:
Got it, we need this since we are updating the options - `$('#to_format').html(options);` Have fixed in my new commit